Makefile

```diff
@@ -79,6 +89,10 @@ set_env_class_stage:
 	$(eval export ENABLE_AUTO_DEPLOY=true)
 	$(eval export DISABLE_AUTODELETE=1)
 	$(eval export PAAS_CF_TAG_FILTER=stage-*)
+	$(eval export ROOT_DNS_ZONE=cloudpipeline.digital)
+	$(eval export APPS_ROOT_DNS_ZONE=cloudpipelineapps.digital)
+	$(eval export ENV_DNS_ZONE=${AWS_ACCOUNT}.${ROOT_DNS_ZONE})
```

Comment:
We can't use `AWS_ACCOUNT` here.  `AWS_ACCOUNT` is set to `stage`, but the domain needs to be `staging.cloudpipeline.digital`